src/interfaces/node.h

```diff
@@ -126,6 +129,12 @@ class Node
     //! Get proxy.
     virtual bool getProxy(Network net, Proxy& proxy_info) = 0;
 
+    //! Get default proxy address.
+    virtual std::string defaultProxyAddress() = 0;
```

Comment:
Are these actually default settings values or just values presented to the user?

Reply:
These are proper default values. I would leave them here for now, but maybe they can be moved into the netbase later.